### PR TITLE
Replace node-sass with (dart) sass

### PR DESCRIPTION
**Pull request by st3v3nhunt**
https://eaflood.atlassian.net/browse/PSD-970

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -10,8 +10,8 @@
   "main": "app/index.js",
   "scripts": {
     "build": "npm run build:css",
-    "build:css": "node-sass --output=app/static/stylesheets --output-style=expanded app/static/stylesheets",
-    "build:css-watch": "node-sass --output=app/static/stylesheets --output-style=expanded --source-map true --watch app/static/stylesheets",
+    "build:css": "sass --load-path=./ app/static/stylesheets",
```
Comment (st3v3nhunt) [inline package.json:13]: `sass` defaults to output in [expanded style](https://github.com/sass/node-sass#outputstyle).
The output directory is optional and the file is written into the same directory.

File: package.json [hunk 2]
```diff
@@ -10,8 +10,8 @@
...
+    "build:css-watch": "npm run build:css -- --watch",
```
Comment (st3v3nhunt) [inline package.json:14]: Rather than duplicating the options for `build:css` I just run the task add pass the `--watch` flag.

File: README.md [hunk 3]
```diff
@@ -24,25 +24,25 @@ Values for development are set in the Docker Compose configuration.
...
-| OKTA_CLIENT_SECRET                    | Client Secret of Okta OpenID Connect app  | no       |                       |                             |
-| OKTA_AUTH_SERVER_ID                   | ID of Okta custom authorisation server    | no       |                       |                             |
-| B2C_CLIENT_ID                         | Client ID of B2C OpenID Connect app       | no       |                       |                             |
-| B2C_CLIENT_SECRET                     | Client Secret of B2C OpenID Connect app   | no       |                       |                             |
-| B2C_URL                               | OAuth URL of B2C OpenID Connect app       | no       |                       |                             |
-| SITE_URL                              | URL of site, i.e. https://mysite.com      | no       |                       |                             |
+| Name                           | Description                              | Required  | Default     | Valid                         | Notes                                                                             |
```
Comment (st3v3nhunt) [inline README.md:27]: There were a couple of comments for `APPINSIGHTS_INSTRUMENTATIONKEY` and `APPINSIGHTS_CLOUDROLE` that were not part of the table. I've added an additional column called `notes` to include them.

**Review by fareedmohammed — APPROVED** [review]
LGTM

**Pull request merged**